RELEASE-NOTES.md

```diff
@@ -1,5 +1,60 @@
 # Prefect Release Notes
 
+## Release 2.16.7
+
+### Introducing `prefect shell` 💻 for observing CLI commands
+You can now observe CLI commands as a Prefect flow. For example, take the command:
+```console
+» curl http://wttr.in/Chicago\?format\=3
+Chicago: ⛅️  +50°F
+```
+
+To run this as a Prefect flow, you can use the following CLI command:
+```python
+» prefect shell watch "curl http://wttr.in/Chicago?format=3"
+17:32:39.562 | INFO    | prefect.engine - Created flow run 'powerful-mushroom' for flow 'Shell Command'
+17:32:40.171 | INFO    | Flow run 'powerful-mushroom' - Chicago: ⛅️  +50°F
+17:32:40.315 | INFO    | Flow run 'powerful-mushroom' - Finished in state Completed()
+```
+
+See the PR for implementation details: https://github.com/PrefectHQ/prefect/pull/11998
```

Comment:
Can you link to the docs George wrote as well? :)

Reply:
like: 
see the docs to learn how to:
- run a shell command as a Prefect flow on-demand with `watch`
- schedule a shell command as a recurring Prefect flow using `serve`

Reply:
good idea! added in [d23b9d3](https://github.com/PrefectHQ/prefect/pull/12476/commits/d23b9d3b9ba871016afe571d22c970877ccbd112)